clear/main.py

```diff
@@ -79,16 +79,14 @@ def get_command(self, ctx, cmd_name):
 # location (configdb?), so that we prevent the continous execution of this
 # bash oneliner. To be revisited once routing-stack info is tracked somewhere.
 def get_routing_stack():
-    command = "sudo docker ps | grep bgp | awk '{print$2}' | cut -d'-' -f3 | cut -d':' -f1"
+    cmd0 = ["sudo", "docker", "ps"]
+    cmd1 = ["grep", "bgp"]
+    cmd2 = ["awk", '{print$2}']
+    cmd3 = ["cut", "-d", '-', "-f3"]
```

Comment:
>"-d", '-' [](http://example.com/codeflow?start=19&length=9)

Originally it is one arg.

Reply:
the result is the same as original behavior, but I did change to one arg.

Reply:
Just wandering, is there any diff between `-d-` and `-d'-'` ?

Reply:
if use direct command in linux, then no difference. But in this case `cmd3 = ["cut", "-d'-'", "-f3"]` will lead to an error, cut: the delimiter must be a single character. So we need to use either `cmd3 = ["cut", "-d-", "-f3"]` or `cmd3 = ["cut", "-d", '-', "-f3"]`

Reply:
Make sense. Bash handles the quotation marks. `shell=False` bypassed the shell.